### PR TITLE
YogStation : Adding decals, fixing stairs, changing library

### PR DESCRIPTION
**Pull request by EIB0N**
#### Read our contributing.md if you haven't already: https://github.com/yogstation13/Yogstation/blob/master/.github/CONTRIBUTING.md

# General Documentation

### Intent of your Pull Request

Making some stylistic changes to areas in the map to make the architecture feel more robust starting with the library. This pr adds a shadow decal that is at the core of the depth illusion. In addition, it also fixes the messed up goon stairs that were not fully there.

### Why is this change good for the game?

Map looks cooler and people look forward to new things

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Library has been remodeled to look cooler 

### What should players be aware of when it comes to the changes your PR is implementing?

Please try not to destroy the turf it kinda messes up the illusion but that goes for any turf with decals, it can be readded like any other decal so it is fine. 

### What general grouping does this PR fall under? 

Remodel Library

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

It can all be added 

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

None

# Changelog

library

:cl:  

tweak: Library Remodeled  

/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Looks surprisingly nice

**Review by nesisknesis — APPROVED** [review]

**Review by patpol4 — COMMENTED** [review]
Is this tested for lighting and issues ingame?

**Comment by EIB0N** [comment]
Yes it has been tested for lighting as well as the disposal bin

**Review by patpol4 — APPROVED** [review]
Map review done over discord dm's. 

Let's see how this goes
